pkg/testing/v1/service.go

```diff
@@ -172,6 +172,18 @@ func WithServiceAccountName(serviceAccountName string) ServiceOption {
 	}
 }
 
+// WithPodSpecDryRunEnabled enables the podspec dry-run feature.
+func WithPodSpecDryRunEnabled() ServiceOption {
```

Comment:
There's already a `WithServiceAnnotation` to save you the trouble of initializing the map etc.